web/src/popover_menus.js

```diff
@@ -441,4 +473,63 @@ export function initialize() {
             message_actions_popover_keyboard_toggle = false;
         },
     });
+
+    // Starred messages popover
+    tippy_no_propagation(".starred-messages-sidebar-menu-icon", {
+        placement: "right",
+        maxWidth: "none",
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "bottom",
+                    },
+                },
+                {
+                    name: "preventOverflow",
+                    options: {
+                        padding: 40,
+                    },
+                },
+            ],
+        },
+        onMount(instance) {
+            const $popper = $(instance.popper);
+            $popper.addClass("starred-messages-popover");
+            starred_messages_popover_instance = instance;
+
+            $popper.one("click", "#unstar_all_messages", () => {
+                starred_messages_ui.confirm_unstar_all_messages();
+                instance.hide();
+            });
+            $popper.one("click", "#toggle_display_starred_msg_count", () => {
+                const data = {};
+                const starred_msg_counts = user_settings.starred_message_counts;
+                data.starred_message_counts = JSON.stringify(!starred_msg_counts);
+
+                channel.patch({
+                    url: "/json/settings",
+                    data,
+                });
+                instance.hide();
+            });
+        },
+        onShow(instance) {
+            popovers.hide_all_except_sidebars();
+            const show_unstar_all_button = starred_messages.get_count() > 0;
+
+            instance.setContent(
+                parse_html(
+                    render_starred_messages_sidebar_actions({
+                        show_unstar_all_button,
+                        starred_message_counts: user_settings.starred_message_counts,
+                    }),
+                ),
+            );
+        },
+        onHidden(instance) {
+            instance.destroy();
```

Comment:
Why doesn't this set  `starred_messages_popover_instance` to undefined here, like the other popover menu with an `_instance` variable does?

Reply:
Hmmm, there doesn't seem to be any reason why it wouldn't be here. I'll fix this.